docusaurus/docs/iOS/uikit/custom-components.md

```diff
@@ -12,11 +12,10 @@ These are the steps that you need to follow to use a custom component:
 1. Make changes to layout, styling, behavior as needed.
 1. Configure the SDK to use your custom component.
 
-To make customizations as easy as possible all view components share the same lifecycle and expose common properties such as `content`, `components` and `appearance`. When building your own custom
-component in most cases you only need to override methods from the `Customizable` protocol such as `updateContent()`, `setUpAppearance()` or `setUpLayout()`.
+To make customizations as easy as possible all view components share the same lifecycle and expose common properties such as `content`, `components` and `appearance`. When building your own custom component if you only want to change the styling and the layout of the view, you only need to override the  `setUpAppearance()` and `setUpLayout()` functions. In case you want to change the logic of the component or how the content is updated, you should override the `updateContent()` function.
```

Comment:
why not include all the docs in this PR?

Reply:
Yes Im working on it at the moment